pkg/injector/injector.go

```diff
@@ -29,6 +30,14 @@ const port = 4000
 
 var log = logger.NewLogger("dapr.injector")
 
+var allowedControllersServiceAccounts = []string{
+	"replicaset-controller",
```

Comment:
keep this mandatory and the others optional (just log debug if error for those), this way it would not fail if cannot get the service account due to a local installation that does not contain those.

Reply:
Changes done : https://github.com/dapr/dapr/pull/2828/files#diff-d5342f32a749b172ddd2344afebd0f1a4ccaea7b4cf01b17f5a24e22677e2a96R112